### PR TITLE
Bump Kubevirt to v0.49.0

### PR DESCRIPTION
**Pull request by guangbochen**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Bump Kubevirt to a newer version of some dependency features, e.g. https://github.com/harvester/harvester/issues/574

**Solution:**
Bump Kubevirt to v0.49.0

**Related Issue:**
https://github.com/harvester/harvester/issues/1783

**Test plan:**
- Verify the CI is passed.
- Run `make build-iso` and has no blockers.
- Install the Harvester with the local ISO build and verify all of the system components are running with `health` status.
- Run regular regress tests, e.g upload an image, create a VM and create a Volume, etc.
- Note: upgrade from v1.0.0 to the newer version will need to be verified via https://github.com/harvester/harvester/issues/1861

Note: please ignore the CodeFactor CI because of an existing format issue.

### CODE REVIEW

File: deploy/charts/harvester/values.yaml [hunk 1]
```diff
@@ -26,8 +26,8 @@ kubevirt-operator:
   containers:
     operator:
       image:
-        repository: registry.suse.com/suse/sles/15.3/virt-operator
-        tag: &kubevirtVersion 0.45.0-8.4.3
+        repository: registry.suse.com/harvester-beta/virt-operator
+        tag: &kubevirtVersion 0.49.0-1
     ## The following images are placeholder for images in use.
     ## They are not used by the kubevirt-operator chart.
     controller:
```
Comment (futuretea) [inline deploy/charts/harvester/values.yaml:33]: The following `registry.suse.com/suse/sles/15.3` also need to be changed to `registry.suse.com/harvester-beta`

Comment (guangbochen) [reply deploy/charts/harvester/values.yaml:33]: Updated, thanks.

**Review by futuretea — APPROVED** [review]
LGTM

**Review by weihanglo — APPROVED** [review]
LGTM!

**Pull request merged**